KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyView.swift

```diff
@@ -19,38 +20,43 @@ struct KHLoyaltyViewID {
     public static let burnPointsSwitch = "burn_points_switch"
     public static let infoView = "info_view"
     public static let infoLabel = "info_label"
+    public static let balanceView = "balance_view"
+    public static let balanceLabel = "balance_label"
 }
 
-final class KarhooLoyaltyView: UIStackView {
-    
-    private let drawAnimationTime: Double = 0.45
-    private let standardSpacing: CGFloat = 12.0
-    private let smallSpacing: CGFloat = 4.0
-    private let mediumCornerRadius: CGFloat = 8.0
-    private let cornerRadius: CGFloat = 3.0
-    private let borderWidth: CGFloat = 1.0
+final class KarhooLoyaltyView: UIView {
     
     private var presenter: LoyaltyPresenter?
     private var didSetupConstraints: Bool = false
     private var topSwitchConstraint: NSLayoutConstraint?
     private var bottomSwitchConstraint: NSLayoutConstraint?
     
     // MARK: - UI
+    private lazy var containerStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.accessibilityIdentifier = KHLoyaltyViewID.containerStackView
+        stackView.spacing = UIConstants.Spacing.medium
+        stackView.axis = .vertical
+        stackView.distribution = .fill
+        return stackView
+    }()
+    
     // Note: The burnPointsSwitch is hidden in case can_burn == false. This stack view makes sure to strech the labels until the right edge of the view in this scenario
     private lazy var loyaltyStackView: UIStackView = {
         let stackView = UIStackView()
         stackView.translatesAutoresizingMaskIntoConstraints = false
         stackView.accessibilityIdentifier = KHLoyaltyViewID.loyaltyStackView
         stackView.axis = .horizontal
-        stackView.spacing = standardSpacing
+        stackView.spacing = UIConstants.Spacing.medium
         stackView.layer.borderColor = KarhooUI.colors.lightGrey.cgColor
-        stackView.layer.borderWidth = borderWidth
-        stackView.layer.cornerRadius = cornerRadius
+        stackView.layer.borderWidth = UIConstants.Dimension.Border.standardWidth
+        stackView.layer.cornerRadius = UIConstants.CornerRadius.small
         stackView.isLayoutMarginsRelativeArrangement = true
-        stackView.directionalLayoutMargins = NSDirectionalEdgeInsets(top: standardSpacing,
-                                                                     leading: standardSpacing,
-                                                                     bottom: standardSpacing,
-                                                                     trailing: standardSpacing)
+        stackView.directionalLayoutMargins = NSDirectionalEdgeInsets(top: UIConstants.Spacing.standard,
```

Comment:
According to all major swift style guides, the preferred formatting of longer declarations should include line breaks after parentheses.
```
stackView.directionalLayoutMargins = NSDirectionalEdgeInsets(
    top: UIConstants.Spacing.standard,
    leading: UIConstants.Spacing.medium,
    bottom: UIConstants.Spacing.medium,
    trailing: UIConstants.Spacing.medium
)
```

Because of that, arguments are always in the same line (considering vertical perspective), no matter on method name. 
This makes code easier to read and follow. Especially in case of multiple-nested scopes.

Reply:
When it comes to long declarations, personally I prefer the line break after the first parameter, with subsequest parameters under the first one (as it already is in the SDK). It provides more visual separation from the content of the method (not applicable in this case with the directional margins declaration, but in general). I find it more readable than condensing everything to the left. Could you add an entry for this proposal on new guideline in your ideas page on Confluence?